distribution/docs/src/main/jdocs/content/_plugins/post-process-plugin-intro-contents.adoc

```diff
@@ -0,0 +1,17 @@
+:type: pluginIntro
+:status: published
+:title: Post-Process Plugins
+:link: _post_process_plugins
+:summary: Performs additional processing after ingest.
+:plugintypes: postprocess
+:order: 06
+
+[NOTE]
+====
+This code is experimental. While this interface is functional and tested, it may change or be removed in a future version of the library.
+====
+
+*Post-Process Plugins* are invoked after a metacard has been created, updated, or deleted and committed to the Catalog.
```

Comment:
I think to set it off as a "type of thing." It's really arbitrary, so we can drop it if you'd like.